src/scss/base/_base.scss

```
@@ -0,0 +1,14 @@
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
+  font-family: $primary-font;
+  color: $text-color;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
```

```suggestion
a,
a:hover {
  text-decoration: none;
}
```
Đồng thời bổ sung endline ở các file